DQMOffline/Trigger/python/SoftMuHardJetMETSUSYMonitor_cff.py

```diff
@@ -5,87 +5,84 @@
 
 from DQMOffline.Trigger.SusyMonitor_cfi import hltSUSYmonitoring
 
-SoftMuHardJetMETSUSYmonitoring = hltSUSYmonitoring.clone()
-SoftMuHardJetMETSUSYmonitoring.FolderName = cms.string('HLT/SUSY/SoftMuHardJetMET/')
-
-SoftMuHardJetMETSUSYmonitoring.numGenericTriggerEventPSet.hltInputTag = cms.InputTag("TriggerResults","","HLT")
-SoftMuHardJetMETSUSYmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring(
-    "HLT_Mu3er1p5_PFJet100er2p5_PFMET70_PFMHT70_IDTight_v*",
-    "HLT_Mu3er1p5_PFJet100er2p5_PFMET80_PFMHT80_IDTight_v*",
-    "HLT_Mu3er1p5_PFJet100er2p5_PFMET90_PFMHT90_IDTight_v*" 
+SoftMuHardJetMETSUSYmonitoring = hltSUSYmonitoring.clone(
+    FolderName = 'HLT/SUSY/SoftMuHardJetMET/',
+    met   = "pfMet",
+    jets  = "ak4PFJetsCHS",
+    muons = "muons",
+    HTdefinition     = 'pt>30 & abs(eta)<2.5',
+    leptJetDeltaRmin = 0.4,
+    MHTdefinition    = 'pt>30 & abs(eta)<2.4',
+    numGenericTriggerEventPSet = dict(hltInputTag = ["TriggerResults","","HLT"],
+                                      hltPaths = ["HLT_Mu3er1p5_PFJet100er2p5_PFMET70_PFMHT70_IDTight_v*",
+                                                  "HLT_Mu3er1p5_PFJet100er2p5_PFMET80_PFMHT80_IDTight_v*",
+                                                  "HLT_Mu3er1p5_PFJet100er2p5_PFMET90_PFMHT90_IDTight_v*" ])
 )
 
-SoftMuHardJetMETSUSYmonitoring.met   = cms.InputTag("pfMetEI")
```

Comment:
About "MET hanlde not valid" message which disappears, this is due to a wrong collection set BEFORE the PR, since Event Interpretation has been removed back in 2021.
This PR corrected above by setting to met   = "pfMet", right above. Since the SoftMuHardJetMETSUSYmonitoring is cloned several times below, it seems it just complained but no crash. Now it should be reading an existing collection.
Thanks

Reply:
Okay, @jfernan2  I will implement those suggestions in this PR. As there were multiple commits and changes over previous commits themselves, there could be some files missed. One such file was pointed out by @perrotta. Thank you for that.